org-mime.el

```diff
@@ -479,16 +480,20 @@ If SUBTREEP is t, curret org node is subtree."
     (message "Warning: org-element-map is not available. File keywords will not work.")
     '())))
 
-(defun org-mime-build-mail-other-headers (cc bcc)
-  "Build mail header from CC and BCC."
+(defun org-mime-build-mail-other-headers (cc bcc from)
+  "Build mail header from CC, BCC, and FROM."
```

Comment:
You need better algorithm for this part. The original code might not be written by me.